spec/models/season/phase_switcher_spec.rb

```diff
@@ -1,35 +1,43 @@
 require 'spec_helper'
 
+describe Season::PhaseSwitcher do
 
-context 'when switching phases' do
-  subject { Season.current }
+  it "is available as described class" do
+    expect(described_class).to eq Season::PhaseSwitcher
+  end
+
+  context 'when switching phases' do
```

Comment:
Since what's happening here is basically testing a `PhaseSwitcher`'s class method, I'd suggest to change this line to:
```ruby
describe '.destined' do
```
this makes things more obvious for people who read the test.